OpenRA.Mods.Common/Widgets/Logic/Editor/ActorEditLogic.cs

```diff
@@ -309,6 +309,28 @@ public override void Tick()
 							slider.OnChange += value => so.OnChange(actor, value);
 							slider.OnChange += value => editorActionHandle.OnChange(value);
 
+							var valueField = sliderContainer.Get<TextFieldWidget>("VALUE");
+
+							Action<float> updateValueField = (float f) =>
```

Comment:
Style nit: no need for the explicit type or parens, so this could be `Action<float> updateValueField = f =>`

Reply:
But i'm not sure that the extra checks actually give us anything (I may be missing something), so this entire block could simplify down to something like:
```csharp
var valueField = sliderContainer.GetOrNull<TextFieldWidget>("VALUE");
if (valueField != null)
{
	Action<float> updateValueField = f => valueField.Text = ((int)f).ToString();
	updateValueField(so.GetValue(actor));
	slider.OnChange += updateValueField;

	valueField.OnTextEdited = () =>
	{
		float result;
		if (float.TryParse(valueField.Text, out result))
			slider.UpdateValue(result);
	};
}
```

Note that i've also changed `valueField` to use `GetOrNull` and added a null check to keep backwards compatibility - this is generally a good practice when working with non-critical UI additions.

Reply:
Right `((int)slider.Value) != ((int)result)` might be uneccessary, i think i added that as a precaution if the value had an initial float value (eg. 5.55), the value would be overwritten to an integer when you load the map

The slider can use and produce float values (depending on its width), **but** i think the only fields that use it are integer: health and facing? I dont think they can be initialized to floating values ?

The changes look fine.

Reply:
Can you update the PR, or would you prefer one of us to force push over the branch?

Reply:
I can update the PR